schema/me/conversation/index.js

```diff
@@ -167,12 +167,36 @@ export const ConversationFields = {
   },
   last_message: {
     type: new GraphQLNonNull(GraphQLString),
+    description: "This is a snippet of text from the last message.",
     resolve: conversation => {
       return get(conversation, "_embedded.last_message.snippet")
     },
   },
   last_message_at: date,
 
+  // This is our desired API for an unread indicator, for a given email:
+  // If the given email address is a recipient of the last message, return true.
+  is_last_message_to: {
+    type: GraphQLBoolean,
+    args: {
+      email_address: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: (conversation, { email_address }) => false, // eslint-disable-line no-unused-vars
+  },
+  // If the given email is a recipient of the last message, return their timestamped
+  // 'read' event, otherwise null.
+  last_message_read_at: {
+    type: GraphQLString,
+    args: {
+      email_address: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: (conversation, { email_address }) => null, // eslint-disable-line no-unused-vars
+  },
```

Comment:
Sorry I'm not sure how to read this... would a query include an `email_address: '...'` param, and the response something like `is_last_message_to: false, last_message_read_at: null` or `is_last_message_to: true, last_message_read_at: {event: 'open', ...}`? A test would clarify this.

Re. naming, `is_last...` is OK assuming we prefer the `is_...` pattern for booleans in this repo. `last_message_read_at` is strange since the event is `open` and the contents are an event rather than a timestamp. Can it be `last_message_open`?